builder/template/config.go

```diff
@@ -36,7 +36,7 @@ func TemplateRootDir() (string, error) {
 
 	tmplPath := filepath.Join(config, "suborbital", "templates")
 
-	if os.Stat(tmplPath); err != nil {
+	if _, err := os.Stat(tmplPath); err != nil {
```

Comment:
Another very tiny fix here is that rather than instantiate and assign the err variable (`:=`), we can skip the instantiate part and only reassign, because a variable with the name `err` (and the same type) already exists.

So instead of `if _, err := os.Stat...`, we can do `if _, err = os.Stat...`. Note the missing `:`. That way we're not going to shadow an earlier declared variable :)